
Fixes complicting declarations of variables ip_optlen in:

globals.h: extern char ip_optlen
and
main.c: unsigned ip_optlen

--- hping3.git.orig/main.c
+++ hping3.git/main.c
@@ -152,7 +152,7 @@ unsigned char
 	lsr		[255] = {0},
 	ssr		[255] = {0};
 
-unsigned
+char
 	ip_optlen	= 0;
 
 struct sockaddr_in
